README.md

```diff
@@ -39,6 +39,9 @@ cds compile <path to srv folder> --to ord [-o] [destinationFilePath]
 2) Check the following relative paths for ORD information - `/.well-known/open-resource-discovery` , `/open-resource-discovery/v1/documents/1`.
 
 
+<img width="1300" alt="Sample Application Demo" style="border-radius:0.5rem;" src="ord.gif">
```

Comment:
ord.gif should be in a separate folder called asset/etc

Reply:
made the changes